src/components/Placeholder/Placeholder.tsx

```diff
@@ -1,24 +1,19 @@
 import * as React from "react";
-import bem from "../../utils/bem";
 
 export interface PlaceholderProps {
-  /** BlockName for use with BEM. See how to work with blockNames and BEM here: http://getbem.com/introduction/ */
-  blockName?: string;
-  /** Modifiers array for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
-  modifiers?: string[];
+  /** Modifier string for the classname. */
+  modifier?: string;
```

Comment:
Should we just call this `variant`?

Reply:
Yeah, that works. Do we need this component long-term?

Reply:
Placeholder is an internal component, so it'll be needed as long as we want to use it.  Right now, it is only used in the stories and tests for `Template`, where I think it is useful.  Now that BEM has been removed, I don't think it needs any further development.